timesketch/tsctl.py

```diff
@@ -49,14 +49,18 @@ def cli():
 
 
 @cli.command(name="list-users")
-def list_users():
+@click.option("--status", "-s", is_flag=True, help="Show status of the users.")
```

Comment:
Can we also get an option to show group memberships of for each user? Some output like this: `user:group1,group2,group3` maybe?

Reply:
done